vmdb/app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -21,3 +21,13 @@
                 {:controller => "ems_container", :action => 'show', :id => @record.ext_management_system.id.to_s},
                 {:title => _("Show this container group's parent %s") % ui_lookup(:table => "ems_container")},
                 '/images/icons/16/link_external.gif')
+        - if role_allows(:feature => "containers")
+          - num_containers = @record.number_of(:containers)
+          - if num_containers == 0
+            %li.disabled
+              = link_to(_('Containers (0)'), "#")
```

Comment:
@abonas yes we should use ui_lookup here and on any other places that are being changed in this PR.